web-app/src/ng1/admin/layers/layer.html

```diff
@@ -19,6 +19,9 @@ <h2>Layer: {{$ctrl.layer.name}}</h2>
           <h2>
             <button class="btn btn-default pull-right" ng-click="$ctrl.editLayer($ctrl.layer)"><i class="fa fa-edit"></i> Edit</button>
           </h2>
+          <h2>
+            <button ng-if="layer.isFileBased" (click)="downloadLayer()">Download</button>
```

Comment:
This appears to be missing the `$ctrl` prefix, so the download button never appears for any layers.